Validation/Configuration/python/globalValidation_cff.py

```diff
@@ -120,11 +120,15 @@
 )
 globalValidationTrackingOnly = cms.Sequence()
 
-def _modifyGlobalValidationForPhase2( theProcess ):
-    theProcess.load('Validation.Configuration.gemSimValid_cff')
-    theProcess.load('Validation.Configuration.me0SimValid_cff')
-    theProcess.globalValidation += theProcess.gemSimValid
-    theProcess.globalValidation += theProcess.me0SimValid
+from Validation.Configuration.gemSimValid_cff import *
+from Validation.Configuration.me0SimValid_cff import *
+
+_run3_globalValidation = globalValidation.copy()
+_run3_globalValidation += gemSimValid
+
+_phase2_globalValidation = _run3_globalValidation.copy()
+_phase2_globalValidation += me0SimValid
 
 from Configuration.StandardSequences.Eras import eras
-modifyConfigurationStandardSequencesGlobalValidationForPhase2_ = eras.phase2_muon.makeProcessModifier( _modifyGlobalValidationForPhase2 )
+eras.run3_GEM.toReplaceWith( globalValidation, _run3_globalValidation )
+eras.phase2_muon.toReplaceWith( globalValidation, _phase2_globalValidation )
```

Comment:
For clarification (and my education), in case both `run3_GEM` and `phase2_muon` sub-eras are active, the order of actions is
- `run3_GEM` replaces `globalValidation` with `_run3_globalValidation`
- `phase2_muon` replaces `globalValidation` with `_phase2_globalValidation`

right? (I'm mainly confused by @slava77's "phase2_muon is executed first")